### PR TITLE
docs: update dev guide

### LINKED ISSUE #477: Update dev docs
### Description
As a developer
I want the commands in the dev guidelines to be up to date
So that I can run the app
### Tasks (How)
- [x] update docs

### PR DESCRIPTION
**Pull request by BCerki**
Fixes #477 

## Proposed Changes

- split dev guide into two sections for clarity; one for getting Storybook running locally, one for building and deploying etc.
- update commands to run Storybook

### CODE REVIEW

File: docs/developer-guide.md [hunk 1]
```diff
@@ -39,34 +61,3 @@ In order to avoid potential side effects using `lerna version/publish` commands,
...
-- Clone this repository
-- Run `yarn` to install package dependencies
-- Run `yarn build` to compile all packages
-- Run `yarn start` in an example app to test
-
-```sh
-yarn bootstrap
-yarn llink
-yarn build
-```
```
Comment (BCerki) [inline docs/developer-guide.md:54]: Couldn't figure out how these two lists of commands were supposed to be used together. I changed it to the instructions Kristen gave me in Notion

File: docs/developer-guide.md [hunk 2]
```diff
@@ -39,34 +61,3 @@ In order to avoid potential side effects using `lerna version/publish` commands,
...
-
-- base
-- gov
-
-To build, run `yarn llink` followed by `yarn build-storybook:<theme>` where theme is one of:
-
-- base
-- gov
-
-To test build locally, run `npx http-server ./storybook-static` after building.
```
Comment (BCerki) [inline docs/developer-guide.md:68]: Not sure these were about. Style changes hot reload without needing to build. But when I tried making a new component I couldn't get it to show in Storybook.

File: docs/developer-guide.md [hunk 3]
```diff
@@ -1,25 +1,56 @@
...
+1. Install `asdf` if not already installed
+2. Clone this repository
+3. `cd` into the repository
+
+### Build and Run Storybook
+
+- Run `yarn` to install package dependencies (follow any prompts `asdf` provides)
+- Run `yarn llink` to set up symlinks
+- Run `yarn build` to compile all packages
+- Run `yarn storybook:<theme>` where `<theme>` is `bcgov`, `button`, or your custom theme
```
Comment (BCerki) [inline docs/developer-guide.md]: These are the instructions Kristen gave me a year ago. We might additionally need `bootstrap`? https://github.com/button-inc/service-development-toolkit/pull/457/commits/fe8808151a04fc1d1352d963818579e0bba73d99

Comment (tmastrom) [reply docs/developer-guide.md]: yeah running `yarn bootstrap` is necessary before `yarn llink`. I also proposed a similar change to the docs [here](https://github.com/button-inc/service-development-toolkit/pull/478/commits/3d3b9e13a9fc5a9b07c767b4bc63f7b63554a986)

**Review by tmastrom — DISMISSED** [review]

**Review by tmastrom — APPROVED** [review]

**Pull request merged**